extensions/peerjs.js

```diff
@@ -0,0 +1,957 @@
+// Name: PeerJS
+// ID: peerjs
+// Description: A port of the PeerJS library for Scratch.
+// By: MikeDEVTheDucklord
+// Original: MikeDEV
+// License: MIT
+
+/*
+    PeerJS Extension for Scratch
```

Comment:
You don't have to include the license of this extension within the extension itself, the `Licence` tag suffices, but it's still fine.